### DISCUSSION TITLE
Using API to add note created from separate program

### DISCUSSION THREAD

**Discussion by rpodcast** [standardnotes/forum#2708] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Not a specific problem with the Standard Notes app itself, but more about the API capabilities.

**Describe the solution you'd like**
I'm interested to see if the API could be leveraged so I could use a separate program to create the text for the note, and be able to leverage an API call to add the note to my Standard Notes account so it can show up in the app.

**Describe alternatives you've considered**
I did a bit of searching and only found this [subreddit post](https://www.reddit.com/r/StandardNotes/comments/7fwzan/is_there_an_api_available/) which points me to using the Standard File API, but I see that it has been [deprecated](http://standardfile.org/) since 2019.

**Additional context**
I use the R language to compile markdown documents (similar to how Jupyter works) and I am hoping I can compile the notes and send them directly to my Standard Notes account, as if I was sending them to a Git repository that was tied to a website.

**Comment by JaspalSuri** [comment]
Thanks for the feedback! I'll pass it along to the team for review. :) 

You could use a third-party program like [this](https://github.com/tannercollin/standardnotes-fs), but it hasn't been updated since May.
